tensorboard/webapp/runs/views/runs_table/sorting_utils_test.ts

```diff
@@ -0,0 +1,284 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {SortingOrder} from '../../../widgets/data_table/types';
+import {parseNumericPrefix, sortTableDataItems} from './sorting_utils';
+
+describe('sorting utils', () => {
+  describe('parseNumericPrefix', () => {
+    it('returns undefined when a non numeric value is provided', () => {
+      expect(parseNumericPrefix('')).toBeUndefined();
+      expect(parseNumericPrefix('foo')).toBeUndefined();
+      expect(parseNumericPrefix('foo123')).toBeUndefined();
+      expect(parseNumericPrefix(NaN)).toBeUndefined();
+    });
+
+    it('returns all leading numbers from a string', () => {
+      expect(parseNumericPrefix('0')).toEqual(0);
+      expect(parseNumericPrefix('123')).toEqual(123);
```

Comment:
Nit: I would prefer one of these had a string value other than '/' after the number